docs/api/misc.md

```diff
@@ -73,3 +74,20 @@ int res0IndexCount();
 ```
 
 Number of resolution 0 **H3** indexes.
+
+## getPentagonIndexes
+
+```
+void getPentagonIndexes(int res, H3Index *out);
+```
+
+All the pentagon **H3** indexes at the specified resolution.
+`out` must be an array of at least size `pentagonIndexCount()`.
+
+## pentagonIndexCount
+
+```
+int pentagonIndexCount();
+```
+
+Number of pentagon **H3** indexes per resolution.
```

Comment:
Nit: Might be worth noting that this is always 12, but provided as a convenience. That particular number is fairly fundamental to the grid system, extremely unlikely to change, and included in the docs, so we probably don't need to pretend it's an implementation detail.